app/database_etl/airtable_records_handler/airtable_records_formatter.py

```diff
@@ -105,3 +120,21 @@ def apply_study_max_estimate_grade(df: pd.DataFrame) -> pd.DataFrame:
                 subset['estimate_grade'] = level
                 continue
     return df
+
+
+def batch_update_airtable_records(records_to_update, field_names):
+    AIRTABLE_API_KEY = os.getenv('AIRTABLE_API_KEY')
+    AIRTABLE_BASE_ID = os.getenv('AIRTABLE_BASE_ID')
+
+    # Create table object
+    table = Table(AIRTABLE_API_KEY, AIRTABLE_BASE_ID, 'Rapid Review: Estimates')
+
+    # Cycle through all the records to update
+    for i, row in records_to_update.iterrows():
+        # For each record, create a dict where the key is the field name and the value is the new field value
+        # If the value is NaN, convert to None because NaN throws error with Airtable API
+        fields = {x: row[airtable_fields_config[x]] if not pd.isna(row[airtable_fields_config[x]]) else None for x in field_names}
```

Comment:
Might be helpful to comment what exactly is going on in this line. We're taking all the field names in Airtable and then mapping them to their values, but we need to get their values by indexing based on their database names... wonder if there's a way to rewrite in terms of database_names and an inverse mapping of airtable_fields_config that is more intuitive, or if a comment would help? had to stare at this for a couple mins to understand

Reply:
Ya sure can comment. I can't really make the code more readable by extracting out variables unless I turn it into a second for loop without the dict comprehension.